src/js/components/stock-movement-wizard/SendMovementPage.jsx

```diff
@@ -351,7 +351,7 @@ class SendMovementPage extends Component {
    * @public
    */
   getFields() {
-    if (this.state.values.origin.type === 'SUPPLIER') {
+    if (this.state.values.origin.type === 'SUPPLIER' || !this.state.values.hasManageInventory) {
```

Comment:
@DominikaHatala this is old file kepnt only fo rebase pursposes. This file is not used right now.

Reply:
Ohhh nvm, this is to develop XD